oe_theme.theme

```diff
@@ -661,9 +660,6 @@ function oe_theme_preprocess_pattern_accordion(array &$variables): void {
         'label' => $item['title'],
         'icon' => [
```

Comment:
We should drop completely the icon parameter here.

Reply:
We should keep the icon for each item, to specify the icon path for the ecl.

Reply:
Yeah true it's coming from our code... Ignore :)